forui/lib/src/theme/theme_data.dart

```diff
@@ -19,13 +19,21 @@ class FThemeData with Diagnosticable {
   /// The card style.
   final FCardStyle cardStyle;
 
+  /// The separator style.
+  final FSeparatorStyle separatorStyle;
+
+  /// The vertical separator style.
+  final FSeparatorStyle verticalSeparatorStyle;
```

Comment:
Should we combine this into a single class similar to what we're planning to do with `FButtonStyles`? For consistency reasons.